// The Firmament project
// Copyright (c) 2013 Malte Schwarzkopf <malte.schwarzkopf@cl.cam.ac.uk>
//

package firmament;

message TaskFinalReport {
  required uint64 task_id = 1;
  optional uint64 start_time = 2;
  optional uint64 finish_time = 3;
  optional uint64 instructions = 4;
  optional uint64 cycles = 5;
  optional uint64 llc_refs = 6;
  optional uint64 llc_misses = 7;
  optional double runtime = 8;
  optional bool reschedule = 9;
}
